frontend/api/resources.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from tastypie.authentication import Authentication
+from tastypie.authorization import Authorization
+from tastypie.resources import ModelResource
+from tastypie.serializers import Serializer
+
+from models import Notification
+
+from django.utils.timezone import is_naive
+
+# Get an instance of a logger
+logger = logging.getLogger('django')
+
+
+class TZAwareDateSerializer(Serializer):
```

Comment:
@mozgovipc here's how to turn the dates into UNIX timestamps on the ListPremier API